docs/api/public_php_api_creating_content.md

```diff
@@ -1,377 +1,151 @@
-
-!!! note
+## Updating content
 
-    The XSD for the internal XML representation can be found in a separate dependency: <https://github.com/ezsystems/ezplatform-xmltext-fieldtype>.
+To update an existing Content item, you need to prepare a [`ContentUpdateStruct`](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.3/eZ/Publish/API/Repository/Values/Content/ContentUpdateStruct.php)
+and pass it to [`ContentService::updateContent`.](https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.3/eZ/Publish/API/Repository/ContentService.php#L276)
+This method works on a draft, so to publish your changes you need to use `ContentService::publishVersion` as well:
 
-You embed an image in your XML, using the `<embed>` tag, providing an image Content ID as the `object_id` attribute.
+``` php
+try {
+    $contentDraft = $this->contentService->createContentDraft($contentInfo);
 
-!!! note "Using a custom format as input"
+    $contentUpdateStruct = $this->contentService->newContentUpdateStruct();
+    $contentUpdateStruct->initialLanguageCode = 'eng-GB';
+    $contentUpdateStruct->setField('name', $newName);
 
-    More input formats will be added later. The API for that is actually already available: you simply need to implement the [`XmlText\Input`](https://github.com/ezsystems/ezplatform-xmltext-fieldtype/tree/master/lib/FieldType/XmlText/Input) interface. It contains one method, `getInternalRepresentation()` that must return an internal XML string. Create your own bundle, add your implementation to it, and use it in your code.
+    $contentDraft = $this->contentService->updateContent($contentDraft->versionInfo, $contentUpdateStruct);
+    $this->contentService->publishVersion($contentDraft->versionInfo);
 
-``` php
-$input = new \My\XmlText\CustomInput( 'My custom format string' );
-$contentCreateStruct->setField( 'body', $input );
+} catch //...
 ```
 
-## Creating a Content Type
-
-!!! note "Full code"
+## Translating content
 
-    <https://github.com/ezsystems/CookbookBundle/blob/master/Command/CreateContentTypeCommand.php>
+Content translations are created per version. Every version by default contains all the existing translations.
 
-Creating a `ContentType` is actually almost more complex than creating Content. It really isn't as common, and does not require the same kind of API as Content.
-
-Let's split the code in three major parts.
+To translate a Content item to a new language, you need to update it and provide a new `initialLanguageCode`:
 
 ``` php
-try
-{
-    $contentTypeGroup = $contentTypeService->loadContentTypeGroupByIdentifier( 'content' );
-}
-catch ( \eZ\Publish\API\Repository\Exceptions\NotFoundException $e )
-{
-    $output->writeln( "content type group with identifier $groupIdentifier not found" );
-    return;
-}
+$contentDraft = $this->contentService->createContentDraft($contentInfo);
 
+$contentUpdateStruct = $this->contentService->newContentUpdateStruct();
+$contentUpdateStruct->initialLanguageCode = $newLanguage;
```

Comment:
as above, lets declare `$newLanguage` and `$newName` or maybe `$translatedName`?